### PR TITLE
Add assemblyai article

### PR DESCRIPTION
**Pull request by bilgeyucel**
Partially fixes https://github.com/deepset-ai/devrel-board/issues/279

* Add a new author page

### CODE REVIEW

File: content/authors/misra-turp/_index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: author
+title: Mısra Turp
+name: Misra Turp
+slug: misra-turp
+position: Developer Educator @AssemblyAI
```
Comment (TuanaCelik) [inline content/authors/misra-turp/_index.md:6]: Maybe we can start adding '(Guest Author)' - next to the full position?

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+**Audio file: Working from home debate 👇**
+
+**Question: "How many people are speaking in this recording?"**
+|Transcription application | Speaker application |
+|--------------------------| --------------------|
+| The transcription does not provide enough information to determine the exact number of people speaking in this recording. | There are three people speaking in this recording. |
+
+## Code explanation
+
+Let’s see now how to build an application with speaker labels using Haystack and AssemblyAI. In this [Colab notebook](https://colab.research.google.com/github/deepset-ai/haystack-cookbook/blob/main/notebooks/using_speaker_diarization_with_assemblyai.ipynb), you will find the code if you’d like to get started quickly.
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:109]:
```suggestion
Let’s now see how to build an application with speaker labels using Haystack and AssemblyAI. In this [Colab notebook](https://colab.research.google.com/github/deepset-ai/haystack-cookbook/blob/main/notebooks/using_speaker_diarization_with_assemblyai.ipynb), you will find the code if you’d like to get started quickly. 
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+```
+
+Next, we will set up the pipeline with the `AssemblyAITranscriber` to generate the speaker-labeled transcription of any audio file we pass.
+
+Check out the [documentation](https://www.assemblyai.com/docs/integrations/haystack) to find out more about the assemblyai-haystack Python package. 
+
+To use this piece of code, make sure you create an [AssemblyAI account](https://www.assemblyai.com/) and get your free API key. You need an [access token from Hugging Face](https://huggingface.co/settings/tokens) to access the free Inference API as well. 
+
+First, initialize the required Haystack components to index documents with embeddings:
+* `InMemoryDocumentStore`: to store your documents without external dependencies or extra setup
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:128]:
```suggestion
* [`InMemoryDocumentStore`](https://docs.haystack.deepset.ai/docs/inmemorydocumentstore): to store your documents without external dependencies or extra setup
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+* `AssemblyAITranscriber`: to create speaker_labels for the given audio file and convert them into documents
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:129]:
```suggestion
* [`AssemblyAITranscriber`](https://haystack.deepset.ai/integrations/assemblyai): to create speaker_labels for the given audio file and convert them into documents
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+* `DocumentSplitter`: to split your documents into smaller chunks
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:130]:
```suggestion
* [`DocumentSplitter`](https://docs.haystack.deepset.ai/docs/documentsplitter): to split your documents into smaller chunks
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+* `SentenceTransformersDocumentEmbedder`: to create embeddings for each document using sentence-transformers models
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:131]:
```suggestion
* [`SentenceTransformersDocumentEmbedder`](https://docs.haystack.deepset.ai/docs/sentencetransformersdocumentembedder): to create embeddings for each document using sentence-transformers models
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+* `DocumentWriter`: to write these documents into your document store
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:132]:
```suggestion
* [`DocumentWriter`](https://docs.haystack.deepset.ai/docs/documentwriter): to write these documents into your document store
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+            "file_path": "/content/Netflix_Q4_2023_Earnings_Interview.mp3",
+            "summarization": None,
+            "speaker_labels": True
+        },
+    }
+)
+```
+
+Next, it is time to set up the retrieval augmentation (RAG) pipeline for speaker labels. For a RAG pipeline, we need:
+* `SentenceTransformersTextEmbedder`: To create an embedding for the user query using sentence-transformers models
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:182]:
```suggestion
* [`SentenceTransformersTextEmbedder`](https://docs.haystack.deepset.ai/docs/sentencetransformerstextembedder): To create an embedding for the user query using sentence-transformers models
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+* `InMemoryEmbeddingRetriever`: to retrieve `top_k` relevant documents to the user query
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:183]:
```suggestion
* [`InMemoryEmbeddingRetriever`](https://docs.haystack.deepset.ai/docs/inmemoryembeddingretriever): to retrieve `top_k` relevant documents to the user query
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+* `PromptBuilder`: to provide a RAG prompt template with instructions to be filled with retrieved documents and the user query
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:184]:
```suggestion
* [`PromptBuilder`](https://docs.haystack.deepset.ai/docs/promptbuilder): to provide a RAG prompt template with instructions to be filled with retrieved documents and the user query
```

File: content/blog/level-up-rag-with-speaker-diarization/index.md [hunk 11]
```diff
@@ -0,0 +1,249 @@
...
+* `HuggingFaceTGIGenerator`: to infer models served through Hugging Face free Inference API or Hugging Face TGI
```
Comment (TuanaCelik) [inline content/blog/level-up-rag-with-speaker-diarization/index.md:185]:
```suggestion
* [`HuggingFaceTGIGenerator`](https://docs.haystack.deepset.ai/docs/huggingfacetgigenerator): to infer models served through Hugging Face free Inference API or Hugging Face TGI
```

**Review by TuanaCelik — APPROVED** [review]

File: content/authors/misra-turp/_index.md (hunk 1 above)
Comment (bilgeyucel) [reply content/authors/misra-turp/_index.md:6]: Then the title becomes too long imo
image

**Pull request merged**